doc/releasenotes/v0.5.0.rst

```diff
@@ -62,13 +76,50 @@ Script Changes
 - A new script, ``scripts/ci/openmc-xs.bash``, that downloads the OpenMC HDF5 cross section library.
 
 
+
 Python API Changes
 ==================
 
 ..
   Describe any changes to the API
 
 
+- Input file format changes:
+
+  - ``depcode['template_inputfile_path']`` → ``depcode['template_inputfiles_path']``
+  - The ``depcode['template_inputfiles_path']`` input variable now has depletion-code-depentent type:
+    
+    - ``str`` for ``Serpent2``
+    - ``dict`` of ``str`` to ``str`` for ``openmc``
+
+  - ``depcode['iter_inputfile']`` → (removed)
+  - ``depcode['iter_matfile']`` → (removed)
+
+.. note:: While the ``iter_inputfile`` and ``iter_matfile`` variables have been removed from the saltproc input file,
+   their presence and usage in ``DepcodeSerpent`` class is unchanged, save for the fact they can no longer be initialized by the user during object creation.
```

Comment:
Are they still essential parts of the ``DepcodeSerpent`` class or are they deprecated and to be removed later? It's worth clarifying here for your future self or future developers.

Reply:
Okay. I cleared this up